### DISCUSSION TITLE
"PUSHOVER PUSH SERVICE" not working

### DISCUSSION THREAD

**Discussion by dogukanaras** [swar/Swar-Chia-Plot-Manager#347] [Q&A]
I have filled in the user_key api_key fields and restarted many times, but I do not have any information.

**Comment by swar** [comment]
@wolfrage76 @Wofl76  might be able to help.

**Comment by dogukanaras** [comment]
@ wolfrage76 @ Wofl76 can you help please

**Comment by dieg0nob** [comment]
same issue here, no errors in logs (INFO), and don't receive any notification
